src/Microsoft.DotNet.Cli.Utils/ExceptionExtensions.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.DotNet.Cli.Utils.ExceptionExtensions
+{
+    internal static class ExceptionExtensions
+    {
+        public static void MarkAsHandled(this Exception e)
+        {
+            Reporter.Verbose.WriteLine($"Ignoring exception: {e.Message.Red()}");
```

Comment:
`MarkAsHandled` doesn't imply `Reporter.Verbose.WriteLine`.

`ReportAsWarning` might be a clearer name in this case.